packages/reference/src/components/MissingEntityCard/MissingEntityCard.tsx

```diff
@@ -1,37 +1,42 @@
-import * as React from 'react';
+import React from 'react';
 
-import { Card, Flex, IconButton, SectionHeading } from '@contentful/f36-components';
+import { EntryCard, IconButton } from '@contentful/f36-components';
 import { CloseIcon } from '@contentful/f36-icons';
 
-import { ContentEntityType } from '../../types';
-import * as styles from './styles';
-
 export function MissingEntityCard(props: {
-  entityType: ContentEntityType;
-  asSquare?: boolean;
+  isDisabled?: boolean;
   isSelected?: boolean;
-  isDisabled: boolean;
   onRemove?: Function;
+  providerName?: string;
 }) {
+  const providerName = props.providerName ?? 'Source';
+  const description =
+    props.providerName === 'Contentful'
+      ? "Content was deleted or archived or you don't have access rights."
```

Comment:
Archived entries don't result in this error, it's either deletion or access right. please update the copy

For example: 
<img width="793" alt="Screenshot 2024-04-08 at 17 09 23" src="https://github.com/contentful/field-editors/assets/12673605/77ed0c08-9543-445d-ac34-5508ebde3951">

Reply:
Thank you for the info, we'll remove archived from the error messages.